torch/utils/data/datapipes/iter/fileopener.py

```diff
@@ -15,6 +15,8 @@ class FileOpenerIterDataPipe(IterDataPipe[Tuple[str, IOBase]]):
             the file is opened by ``open()``. It defaults to ``b`` which
             means open for reading in binary mode. Another option is
             to use ``t`` for text mode
+        encoding: An optional string that specifies the encoding of the
+            underlying file. It defaults to ``utf-8``.
```

Comment:
I'm not actually sure defaulting to "utf-8" is a great idea -- it seems like maybe the better way is to default to `None` and if we're not reading in binary mode then let `open` resolve it if a user hasn't specified it. This is consistent with what's happening now so it should be transparent from a user perspective.